LocalDynamoDb/LocalDynamo.cs

```diff
@@ -79,7 +81,7 @@ public void Stop()
 
         private AmazonDynamoDBClient CreateClient()
         {
-            var config = new AmazonDynamoDBConfig { ServiceURL = String.Format("http://localhost:{0}", _port) };
+            var config = new AmazonDynamoDBConfig { ServiceURL = Invariant($"http://localhost:{_port}")};
```

Comment:
Nice, I'm going to be stealing this!

Reply:
Yeah, saw it a couple of weeks ago and stole it from somewhere else 😄